src/js/interface.js

```diff
@@ -456,6 +459,16 @@ window.addEventListener("load", function() {
                 if (value===10) {
                   return document.createElement("br");
                 } else {
+                  if(value>=-10240 && value<=-9217) {
+                    convertOutput.highSurrogate = value;
+                    return document.createTextNode("");
```

Comment:
You could also return `null`, and check for nullity in two places where `convertOutput` is used, and if it is null, then don't append any child.

Reply:
If that's how the embedding code works, that's great. I didn't realize the embedded code could handle that case and thought returning an empty text node would be the solution with the least amount of code to change.

Reply:
You would need to change two places that calls `convertOutput` so that it checks for `null`, since for any element, `element.appendChild(null)` will throw `TypeError: Node.appendChild: Argument 1 is not an object.`.

In file [`src/js/interface.js`](https://github.com/MARIE-js/MARIE.js/blob/8e9b06aa961df89be403e2237c1d66eea3bf3a39/src/js/interface.js):
* Line 488: [`outputLog.appendChild(convertOutput(outputList[i]));`](https://github.com/MARIE-js/MARIE.js/blob/8e9b06aa961df89be403e2237c1d66eea3bf3a39/src/js/interface.js#L488)
* Line 805: [`outputLog.appendChild(convertOutput(value));`](https://github.com/MARIE-js/MARIE.js/blob/8e9b06aa961df89be403e2237c1d66eea3bf3a39/src/js/interface.js#L805)

If `convertOutput` returns `null`, then do not call `appendChild`.